### PR TITLE
8847 vre chapter31 scaffolding

### PR DESCRIPTION
**Pull request by rianfowler**

### CODE REVIEW

**Comment by rianfowler** [comment]
Runs and routes.

File: src/js/vre/chapter31/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    this.props.router.push(this.props.route.pageList[1].path);
+  }
+  render() {
+    return (
+      <div className="schemaform-intro">
+        <FormTitle title="Apply for VR&E"/>
+        <SaveInProgressIntro
+          buttonOnly
+          pageList={this.props.route.pageList}
+          startText="Start the Chapter 36 application"
```
Comment (jbalboni) [inline src/js/vre/chapter31/components/IntroductionPage.jsx]: Could change this to 31.

File: src/js/vre/chapter31/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+class ConfirmationPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isExpanded: false };
+  }
+
+  componentDidMount() {
+  }
+
+  toggleExpanded = (e) => {
```
Comment (jbalboni) [inline src/js/vre/chapter31/containers/ConfirmationPage.jsx]: You can remove the expanded related code, I don't think we'll use that.

File: src/js/vre/chapter31/labels.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export const dischargeTypeLabels = {
```
Comment (jbalboni) [inline src/js/vre/chapter31/labels.js]: We could move these up a folder and use them in both forms, probably.

**Review by jbalboni — APPROVED** [review]
LGTM

File: src/js/vre/chapter31/labels.js (hunk 3 above)
Comment (cehsu) [reply src/js/vre/chapter31/labels.js]: Would it make sense to follow the edu form structure and use a utils folder to hold labels, etc?

**Pull request merged**